etc/picongpu/jureca-jsc/gpus_picongpu.profile.example

```diff
@@ -10,55 +10,65 @@ export MY_MAIL="someone@example.com"
 export MY_NAME="$(whoami) <$MY_MAIL>"
 
 # Project Information ######################################## (edit this line)
-#   - project account for computing time
-export proj=$(groups | awk '{print $5}')
+#   - project and account for allocation
+#   jutil user projects will return a table of project associations.
+#   Each row contains: project,unixgroup,PI-uid,project-type,budget-accounts
+#   We need the first and last entry.
+#   Here: select the last available project.
+export proj=$( jutil user projects --noheader | awk '{print $1}' | tail -n 1 )
+export account=$(jutil user projects -n | awk '{print $NF}' | tail -n 1)
 
 # Text Editor for Tools ###################################### (edit this line)
 #   - examples: "nano", "vim", "emacs -nw", "vi" or without terminal: "gedit"
 #export EDITOR="nano"
 
 # Set up environment, including $SCRATCH and $PROJECT
-jutil env activate -p $proj
+# Handle a case where the budgeting account is not set.
+if [ "$account" = "-" ]; then
+    jutil env activate --project $proj;
+else
+    jutil env activate --project $proj --budget $account
+fi
 
 # General modules #############################################################
 #
 module purge
-module load GCC/7.3.0
-module load CUDA/9.2.88
+module load Stages/2024
+module load GCC/12.3.0
+module load CUDA/12
 module load CMake
-module load MVAPICH2/2.3-GDR
-module load Python/3.6.6
-
-# Other Software ##############################################################
+module load OpenMPI/4.1.5
+module load Python/3.11.3
+module load ADIOS2/2.9.2
+module load HDF5/1.14.2
+module load Boost/1.82.0
```

Comment:
No FFTW module - but tha's fine with me.

Reply:
Since shadowgraphy is not used, not including FFTW should reduce compile time, hence the omission.

Reply:
Thanks for the update.